qdrant/v0.10.x/collections.md

```diff
@@ -137,7 +137,7 @@ client.update_collection(
 )
 ```
 
-This command enables indexing for segments that have more than 10000 vectors stored.
+This command enables indexing for segments that have more than 10000 KBs.
```

Comment:
Nitpick, but maybe this is better:

```suggestion
This command enables indexing for segments that have more than 10000 KB of vectors stored.
```

Reply:
That's better, thanks! Will update it quickly